saladinjake/Fast-Food-Fast
# Api version

What does this PR do?
[Pull Request made to elicit review and feed back from collaborators]

http://fastfoodfast-apiservice.herokuapp.com

Description of Task to be completed?
 Completed Task : Api End Point Functionality Test to
○	Place a new order for food.

○	Get a list of orders.

○	Fetch a specific order.

○	Update the order status

 
GET http://fastfoodfast-apiservice.herokuapp.com/orders/:id.

GET | http://fastfoodfast-apiservice.herokuapp.com/orders/:id
 
POST http://fastfoodfast-apiservice.herokuapp.com /:id  Place a new order.

PUT |http://fastfoodfast-apiservice.herokuapp.com /orders/:id  | Update the status of an order.
 
API using url versioning :http://fastfoodfast-apiservice.herokuapp.com/api/v1/orders

 Integrate TravisCI for Continuous Integration in your repository (with ReadMebadge).

Integrate test coverage reporting (e.g. Coveralls) with badge in the ReadMe.

Obtain CI badges (e.g. from Code Climate and Coveralls) and add to ReadMe.
How should this be manually tested?
Ensure to npm install the packages from command line and run the command npm run test to see out put to the console

What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/n/projects/2193654

On Pivotal Tracker, I created user stories to setup and test API endpoints that do the
following using data structures:
○ Place a new order for food.
○ Get a list of orders.
○ Fetch a specific order.
○ Update the order status.
Users can create an account and log in
A user should be able to order for food
The admin should be able to add, edit or delete the fast-food items
The admin should be able to see a list of fast-food items
The admin user should be able to do the following:
a. See a list of orders
b. Accept and decline orders
c. Mark orders as completed
A user should be able to see a history of ordered food
Any background context you want to add?
![capture](https://user-images.githubusercontent.com/26296603/46136759-3c4a5500-c240-11e8-83e0-536af394dddc.PNG)
